private-templates-service/client/src/components/Common/Tags/Tags.tsx

```diff
@@ -25,21 +24,14 @@ const mapStateToProps = (state: RootState) => {
   }
 }
 
-const mapDispatchToProps = (dispatch: any) => {
-  return {
-    updateTags: (tags: string[]) => {
-      dispatch(updateTemplate(undefined, undefined, undefined, undefined, undefined, undefined, tags))
-    }
-  }
-}
-
 interface Props {
   tags?: string[];
   allowEdit?: boolean;
   allowAddTag?: boolean;
   templateID?: string;
   template?: Template;
   updateTags: (tags: string[]) => void;
+  tagRemove: (tag: string) => void;
```

Comment:
You could make these both optional so that you dont need to pass in trivial functions